### PR TITLE
chore: getting rid of some compiler warnings

### PR DESCRIPTION
**Pull request by Sean-Tedrow-LB**
Signed-off-by: Sean-Tedrow-LB <sean.tedrow@launchbadge.com>

### CODE REVIEW

File: sdk/src/main/java/com/hedera/hashgraph/sdk/ChunkedTransaction.java [hunk 1]
```diff
@@ -115,20 +121,21 @@ public int getMaxChunks() {
         return transactionHashes;
     }
 
-    @FunctionalExecutable(type = "java.util.List<TransactionResponse>")
+    @Override @FunctionalExecutable(type = "java.util.List<TransactionResponse>")
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/ChunkedTransaction.java:124]: Can we have the annotations on different lines; this would be more clear.
```java
@Override
@FunctionalExecutable(...)
```

File: sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java [hunk 2]
```diff
@@ -80,7 +80,7 @@ public SdkRequestT setNodeAccountIds(List<AccountId> nodeAccountIds) {
 
     abstract CompletableFuture<Void> onExecuteAsync(Client client);
 
-    @FunctionalExecutable
+    @Override @FunctionalExecutable
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java:83]: Can we put annotation on separate lines.

File: sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java [hunk 3]
```diff
@@ -5,10 +5,12 @@
 
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
 
 abstract class ManagedNode {
     String address;
-    ManagedChannel channel;
+    @Nullable
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java:12]: Personal preference, but can you add a new line before and after the field with an annotation; I think that's more readable that way
```java
String address;

@Nullable
ManagedChannel channel = null;

final ExecutorService executor;
```

File: sdk/src/main/java/com/hedera/hashgraph/sdk/MirrorNetwork.java [hunk 4]
```diff
@@ -2,8 +2,12 @@
 
 import org.threeten.bp.Duration;
 import org.threeten.bp.Instant;
+import com.google.errorprone.annotations.Var;
 
-import java.util.*;
+//import java.util.*;
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/MirrorNetwork.java:7]: Can you remove this line?

File: sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java [hunk 5]
```diff
@@ -3,14 +3,25 @@
 import com.google.errorprone.annotations.Var;
 import com.google.protobuf.ByteString;
 import com.google.protobuf.InvalidProtocolBufferException;
-import com.hedera.hashgraph.sdk.proto.*;
+//import com.hedera.hashgraph.sdk.proto.*;
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java:6]: We can remove this now.

File: sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionId.java [hunk 6]
```diff
@@ -1,18 +1,15 @@
 package com.hedera.hashgraph.sdk;
 
-import com.google.protobuf.ByteString;
+
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionId.java:3]: Very much a nit pick, but can we keep it to one new line between `package ...` and the first `import`?

**Review by danielakhterov — CHANGES_REQUESTED** [review]
Just left some minor nit picks on code style.

File: sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java (hunk 5 above)
Comment (Sean-Tedrow-LB) [reply sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java:6]: I need to start looking at my diffs before I commit

File: sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java [hunk 7]
```diff
@@ -28,15 +31,8 @@ synchronized ManagedChannel getChannel() {
             return channel;
         }
 
-        var channelBuilder = ManagedChannelBuilder.forTarget(address);
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java:31]: This is my bad call. I thought we can remove the conditional on if address ends with `:50212` or `:433`, but after the weekly meeting I realized this is meat to stay in the SDK; so can you undo this change?

**Review by danielakhterov — CHANGES_REQUESTED** [review]

**Review by danielakhterov — APPROVED** [review]
